src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -37,6 +37,13 @@
 ## Version 1.7.0
 * Introduced support for client encryption key resource management required for CosmosDB Client-Side Encryption by adding support for creating, updating and retrieving client encryption keys with following cmdlets: `Get-AzCosmosDbClientEncryptionKey`, `New-AzCosmosDbClientEncryptionKey` and `Update-AzCosmosDbClientEncryptionKey`
 
+## Version 1.6.0
```

Comment:
please move release notes under line 21

Reply:
should I bump version from latest 1.9.0 to 1.9.1?

Reply:
we will calculate the correct version during release, no need to specify one in code changes